app/assets/stylesheets/govuk_publishing_components/components/_title.scss

```diff
@@ -32,11 +32,20 @@
 }
 
 @include govuk-media-query($media-type: print) {
+  .gem-c-title {
+    margin: 5mm 0 10mm !important; // stylelint-disable-line declaration-no-important
+  }
+
   .gem-c-title__context {
     margin: 0;
   }
 
   .gem-c-title__text {
     margin-top: 0;
   }
+
+  .gem-c-title--inverse {
+    color: $govuk-print-text-colour;
+    background-color: unset;
```

Comment:
I don't think we need to unset the background colour here - the component doesn't set one, it's handled by whatever element it is placed into.

Reply:
OK, removed.